examples/petsc/makefile

```diff
@@ -22,7 +22,7 @@ MFEM_LIB_FILE = mfem_is_not_built
 -include $(CONFIG_MK)
 
 SEQ_EXAMPLES =
-PAR_EXAMPLES = ex1p ex2p ex3p ex4p ex5p ex6p ex9p ex10p
+PAR_EXAMPLES = ex1p ex2p ex3p ex4p ex5p ex6p ex9p ex10p ex11p ex28p
```

Comment:
These examples should only be added to this list when MFEM_USE_SLEPC is YES.  I imagine something similar may be needed in the CMake build files.

Reply:
Fixed (both in makefile and CMake build file).